tmeister/static/src/toggles/table/toggle-table.helpers.spec.js

```diff
@@ -114,7 +121,7 @@ describe('toggle-table.helpers.js', () => {
 				},
 			];
 
-			expect(getEnvList(toggles)).toEqual(["production", "stage"]);
+			expect(getEnvList(toggles)).toEqual(["production", "stage", "nonproduction"]);
```

Comment:
After changing the test I was worried that "Production always first" was relying on alphabetical-ness, so I added another environment that would come before the list if it was purely alphabetical.